From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Thu, 25 Nov 2010 09:58:05 +0100
Subject: [PATCH] turn off XATTR support by default

it can be turned on by defining XATTR_SUPPORT=1 on the make command line.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 squashfs-tools/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/squashfs-tools/Makefile b/squashfs-tools/Makefile
index 8d2fa35..d409c17 100644
--- a/squashfs-tools/Makefile
+++ b/squashfs-tools/Makefile
@@ -64,7 +64,7 @@ COMP_DEFAULT = gzip
 # If your C library or build/target environment doesn't support XATTRs then
 # comment out the next line to build Mksquashfs and Unsquashfs without XATTR
 # support
-XATTR_SUPPORT = 1
+#XATTR_SUPPORT = 1
 
 # Select whether you wish xattrs to be stored by Mksquashfs and extracted
 # by Unsquashfs by default.  If selected users can disable xattr support by
-- 
1.7.2.3

